addons/website/models/website.py

```diff
@@ -127,9 +127,16 @@ def _onchange_language_ids(self):
             self.default_lang_id = language_ids[0]
 
     def _compute_menu(self):
-        Menu = self.env['website.menu']
         for website in self:
-            website.menu_id = Menu.search([('parent_id', '=', False), ('website_id', '=', website.id)], order='id', limit=1).id
+            menu_ids = self.env['website.menu'].search([('website_id', '=', website.id)])
+            for menu_id in menu_ids:
```

Comment:
`menu_ids` → `menus`
`menu_id` → `menu`

Reply:
comment: "use field parent_id (1 query) to determine field child_id (2 queries by level)"